cs/configuring.texy

```diff
@@ -292,16 +292,55 @@ Lze konfigurovat některé parametry [Tracy |tracy:] a nastavovat panely do Debu
 
 /--neon
 	tracy:
-		email: %webmasterEmail%
+		# email, na který se posílají notifikace, že došlo k chybě
+		email: dev@example.com
+		fromEmail: robot@example.com
+
+		# ve vývojovém režimu zobrazí chyby typu notice nebo warning jako BlueScreen
 		strictMode: true
-		editor: ...
+		
+		# zobrazí umlčené (@) chybové hlášky
+		scream: true
+
+		# formát odkazu pro otevření v editoru
+		editor: editor://open/?file=%file&line=%line
+
+		# cesta k otevření prohlížeče s uloženým BlueScreen
 		browser: ...
-		bar: # panely do Debugger baru
-			- Nette\Bridges\DITracy\ContainerPanel # alias DI Container baru
+
+		# pro jaké úrovně chyb (E_WARNING, E_ALL, ...) se loguje i BlueScreen
+		logSeverity: 
+
+		# cesta k šabloně s vlastní stránkou pro chybu 500
+		errorTemplate: ...
+
+		# zobrazí Debugger Bar na spodu stránky
+		showBar: true
+
+		# maximální délka řetězce vypisovaná funkcí dump
+		maxLength: 150
+
+		# kolik úrovní se má vypisovat funkcí dump
+		maxDepth: 3
+
+		# zobrazí cestu volání funkce dump
+		showLocation: false
+
+		# nahrazuje originální cesty k souborům a složkám v BlueScreen
+		editorMapping:
```

Comment:
nenapadá mě, jak to říct lépe, takže navrhuji to ponechat bez popisky.

Reply:
Dobre, ono dost pomuze ten priklad s kodem.